src/contexts/graphql-apollo-provider.js

```diff
@@ -0,0 +1,27 @@
+import { ApolloClient,InMemoryCache,ApolloProvider,HttpLink,from } from "@apollo/client";
+import { onError} from '@apollo/client/link/error'
+
+const errorLink = onError(({graphqlErrors,networkError})=>{
+  if(graphqlErrors){
+    graphqlErrors.forEach(({message,location,path})=>{
+      alert(`Graphql Error: ${message}`)
+    })
+  }
+})
+
+const urltohit = from([
```

Comment:
This is improper variable naming convention. I'm sure you know this, but just pointing it out anyway.